### ISSUE TITLE
Special symbols do not work in "Search Playlist"

### ISSUE THREAD

**Issue by justjakka** [fooyin/fooyin#398]
When searching for tags with special symbols in a library selection playlist no results are shown.
for example, searching in library selection playlist with all music selected yeilds nothing:
![image](https://github.com/user-attachments/assets/847bd45f-20d7-486e-bd65-b63802a88337)

**Comment by lstrsrt** [comment]
`:` is recognized as a special token by the parser, but you can escape it (and other special chars) with a backslash. So in this example, `I\:GOR` should work.

**Comment by ludouzi** [comment]
Yeah, `:` is reserved for the HAS/contains operator. Such tokens can be escaped with `\`, or the entire search can be surround by quotes i.e. `"I:GOR"`.
